openquake/commonlib/calculators/hazard.py

```diff
@@ -165,9 +166,11 @@ def export_curves(self, curves, fmt, fname):
             oq.imtls, oq.investigation_time)
         if getattr(oq, 'hazard_maps', None):
             hmaps = self.hazard_maps(curves)
+            # a dictionary IMT -> matrix(P, N)
             saved += export(
                 ('hazard_curves', fmt), export_dir,
-                fname.replace('curve', 'map'), self.sitecol, hmaps,
+                fname.replace('curve', 'map'), self.sitecol,
+                {k: v.T for k, v in hmaps.iteritems()},
```

Comment:
Actually the comment was there but it did not appear in the diff, it is one line up :-(